qiskit_ibm_runtime/base_runtime_job.py

```diff
@@ -135,11 +135,23 @@ def cancel_result_streaming(self) -> None:
             return
         self._ws_client.disconnect(WebsocketClientCloseCode.CANCEL)
 
+    def usage(self) -> Dict[str, Any]:
+        """Return job usage in seconds."""
+        try:
+            metrics = self._api_client.job_metadata(self.job_id())
+            return metrics.get("usage")
+        except RequestsApiError as err:
+            raise IBMRuntimeError(f"Failed to get job metadata: {err}") from None
+
     def metrics(self) -> Dict[str, Any]:
         """Return job metrics.
 
         Returns:
-            Job metrics, which includes timestamp information.
+            A dictionary with job metrics including but not limited to the following:
+
+            * ``timestamps``: Timestamps of when the job was created, started running, and finished.
+            * ``usage``: Job usage metrics
+
```

Comment:
Should we keep this intentionally vague so we have more freedom to update the information returned from the server?

Reply:
This is the definition of `usage` from our docs:

> Usage is a measurement of the amount of time the QPU is locked for your workload.